### DISCUSSION TITLE
Relay module configuration

### DISCUSSION THREAD

**Discussion by chellea** [kizniche/Mycodo#1343] [General]
I am looking for guidance on connecting a relay module.  When I attempt to turn the relay on or off, I get the following

**Cannot manipulate Output 9888c848-3ca0-4c3d-bf5b-e762813c35d8: Output not set up.**

Not sure what I am missing there.

I have tried it with both the Grove 8 channel relay as well as the sainsmart 8 channel relay.  Any guidance would be greatly appreciated.

Thank you.

**Comment by kizniche** [comment]
Please post a diagram or photo of how the relay is wired.

**Reply by chellea** [reply]
Actually, I think I found my issue, the relay board I was trying to use is GPIO...

Can I use this or do I need i2c?

Thanks
